lib/packBuffers.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+
+module.exports = packBuffers;
+
+/**
+ * Repacks the accessed buffer data into contiguous chunks.
+ * Also has the effect of un-interleaving interleaved accessors.
```

Comment:
OK, perhaps name this `uninterleaveAndPackBuffers` (generally, putting "and" in a name is not ideal, but I think it is valuable here) since other stages will want to depend on it for the uninterleaving part.